observations/obs_converters/ARVOR/readme.rst

```diff
@@ -0,0 +1,137 @@
+ARVOR (Profiling Floats)
+========================
+
+This utility converts in-situ profiling float (T/S profile) ASCII files 
+into a DART observation sequence (`obs_seq`) file.
+
+.. contents::
+   :depth: 2
+   :local:
+
+Overview
+--------
+Profiling floats such as ARVOR-C and ARVOR-I (ice-capable) record seawater temperature and 
+salinity at multiple depths during dives and ascents.  
+This converter ingests ASCII (.csv format) files of these profiles and 
+writes them into a DART obs_seq format for assimilation with DART. The 
+csv files are expected to have on the first line a header with 
+different fields names followed by data entries. 
+
+The converter reads the following from the raw data: 
+ * ``dat`` -- timestamp
+ * ``lat`` -- latitude in degrees
+ * ``lon`` -- longitude in degrees
+ * ``pres`` -- pressure in dbar converted to depth in meters
+ * ``sea_water_temperature`` -- temperature in Kelvin, converted to C.
+ * ``salinity`` -- salinity in psu 
+
+.. note::
+
+   **Time handling:** The converter reads the ``dat`` column, parses the timestamp, and
+   constructs a DART ``time_type`` using the Gregorian calendar. The 
+   expected time format is ``YYYY-MM-DDThh:mm:ssZ``
+
+Build & Run
+-----------
+Build as usual with DART converters. Then run, ``./arvor_to_hf``, with an 
+`input.nml` that includes a namelist section that looks like:
+
+.. code-block:: fortran
+
+   &arvor_to_obs_nml
+      file_in           = 'ARVOR_20251006_300534063313500.csv',
+      file_list         = '',               ! path to list of files (or '')
+      file_out          = 'obs_seq.arvor',
+      obs_error_temp    = 0.02,             ! temperature error standard deviation (C)
+      obs_error_sal     = 0.02,             ! salinity error standard deviation (PSU)
+      avg_obs_per_file  = 500000,           ! pre-allocation hint
```

Comment:
i'd say this is more than a 'hint' because i don't see anywhere that the converter can recover if there are more obs than were originally allocated for.   maybe use 'limit' instead of 'hint'?